sdk/storagesync/Microsoft.Azure.Management.StorageSync/src/Generated/SdkInfo_MicrosoftStorageSync.cs

```diff
@@ -19,24 +19,26 @@ internal static partial class SdkInfo
           {
               return new Tuple<string, string, string>[]
               {
-                new Tuple<string, string, string>("StorageSync", "CloudEndpoints", "2019-06-01"),
-                new Tuple<string, string, string>("StorageSync", "OperationStatus", "2019-06-01"),
-                new Tuple<string, string, string>("StorageSync", "Operations", "2019-06-01"),
-                new Tuple<string, string, string>("StorageSync", "RegisteredServers", "2019-06-01"),
-                new Tuple<string, string, string>("StorageSync", "ServerEndpoints", "2019-06-01"),
-                new Tuple<string, string, string>("StorageSync", "StorageSyncServices", "2019-06-01"),
-                new Tuple<string, string, string>("StorageSync", "SyncGroups", "2019-06-01"),
-                new Tuple<string, string, string>("StorageSync", "Workflows", "2019-06-01"),
+                new Tuple<string, string, string>("StorageSync", "CloudEndpoints", "2020-03-01"),
```

Comment:
Please also update the api tags in `AzSdk.RP.props` whenever you generate SDK from a new API version

Reply:
Thanks for reviewing! I've made the change you suggested.